groundstation/backend_api/user.py

```diff
@@ -42,6 +45,52 @@ def get(self, user_id, local_args=None):
         response_object['data'] = user.to_json()
         return response_object, 200
 
+    @create_context
+    @login_required
+    def patch(self, user_id, local_data=None):
+        """ Endpoint for patching a specific user
+
+        :param int user_id: The id of the user to patch
+        :param json_string local_data: This should be used in place of the POST body that would be used through HTTP, used for local calls.
+
+        :returns: response_object, status_code
+        :rtype: tuple (dict, int)
+        """
+        if not local_data:
+            post_data = request.get_json()
+        else:
+            post_data = json.loads(local_data)
+
+        user = User.query.filter_by(id=user_id).first()
+
+        if user is None:
+            response_object = {'status': 'fail', 'message': 'User does not exist'}
+            return response_object, 404
+        
+        try:
+            validated_data = self.validator.load(post_data) # update validator for user
```

Comment:
If "update validator for user" is a todo then recommend the following change. Otherwise, can ignore.

```suggestion
            # TODO: update validator for user
            validated_data = self.validator.load(post_data)
```

Reply:
Oh that comment was left as a todo for myself that I forgot to remove after.